app/src/Readme.tsx

```diff
@@ -3,9 +3,35 @@ import { Navbar } from './Navbar'
 
 export function Readme({ currentPage }: any) {
   return (
-    <div>
-      <Navbar currentPage="#/readme" />
-      Readme...
+    <div className="p-3">
+      <Navbar currentPage="#/" />
+      <h2>What is it?</h2>
+      <p>Nebula is a tool for easing development of Soroban smart contracts.</p>
```

Comment:
I would elaborate a bit more here. What are the parts we are going to see here ? contracts library + wizard ?

Reply:
I would also mention the intention of making it an UI pluggable module. I considered that a good idea.

Reply:
FYI, You can suggest this way:
````
```suggestion
      <p>Nebula is a tool for easing development of Soroban smart contracts. (Some change anywhere)</p>
```
````